README.md

```diff
@@ -106,3 +106,17 @@ To track exceptions through Sentry, configure the `SENTRY_DSN` environment varia
 ```
 SENTRY_DSN=https://aaa:bbb@sentry.io/123 rails s
 ```
+
+
+## Using Basic Auth Instead of DFE Sign-In
+
+For local development, you can disable reliance on DFE Sign-In by creating a
+file `config/settings/development.local.yml` with the contents:
+
+```yaml
+authorised_users:
```

Comment:
```suggestion
authorised_user:
```
singular?


```yml
authorised_user:
 email:
 first_name:
 last_name:
 insecure_password:

```

Reply:
I was thinking we could have multiple users there, but I like your idea of adding the `first_name` and `last_name`, so ya, will have a look at changing that.